client/me/purchases/cancel-private-registration/style.scss

```diff
@@ -23,4 +23,8 @@
 
 .cancel-private-registration__cancel-button {
 	margin-top: 10px;
+
+	&.disabled, &[disabled] {
```

Comment:
Do we really need to specify the `&.disabled` selector? As far I can see it's never used by our markup.